src/problems/MetalAdvectionProblem/test_sne.cpp

```diff
@@ -0,0 +1,558 @@
+    -8.92202532, -8.92173815, -8.92160118, -8.92153673, -8.92150669, -8.9214927,  -8.92148604, -8.92148266, -8.92148075, -8.92147948, -8.92147848, -8.92147761,
+    -8.9214768,	 -8.92147602, -8.92147525, -8.9214745,	-8.92147377, -8.92147304, -8.92147233, -8.92147163, -8.92147094, -8.92147026, -8.92146959, -8.92146894,
+    -8.92146829, -8.92146765, -8.92146703, -8.92146641, -8.92146581, -8.92146521, -8.92146462, -8.92146405, -8.92146348, -8.92146293, -8.92146238, -8.92146185,
+    -8.92146132, -8.9214608,  -8.92146029, -8.92145979, -8.9214593,  -8.92145882, -8.92145835, -8.92145789, -8.92145744, -8.92145699, -8.92145655, -8.92145612,
+    -8.92145569, -8.92145528, -8.92145487, -8.92145447, -8.92145407, -8.92145369, -8.92145331, -8.92145294, -8.92145257, -8.92145221, -8.92145186, -8.92145152,
+    -8.92145118, -8.92145085, -8.92145052, -8.9214502,	-8.92144989, -8.92144959, -8.92144929, -8.921449,   -8.92144871, -8.92144844, -8.92144816, -8.9214479,
+    -8.92144764, -8.92144738, -8.92144713, -8.92144689, -8.92144665, -8.92144642, -8.92144619, -8.92144596, -8.92144574, -8.92144553, -8.92144531, -8.92144511,
+    -8.9214449,	 -8.9214447,  -8.92144451, -8.92144431};
+AMREX_GPU_MANAGED amrex::GpuArray<amrex::Real, 100> z_data{
+    6.08467742e+19, 1.82540323e+20, 3.04233871e+20, 4.25927419e+20, 5.47620968e+20, 6.69314516e+20, 7.91008065e+20, 9.12701613e+20, 1.03439516e+21,
+    1.15608871e+21, 1.27778226e+21, 1.39947581e+21, 1.52116935e+21, 1.64286290e+21, 1.76455645e+21, 1.88625000e+21, 2.00794355e+21, 2.12963710e+21,
+    2.25133065e+21, 2.37302419e+21, 2.49471774e+21, 2.61641129e+21, 2.73810484e+21, 2.85979839e+21, 2.98149194e+21, 3.10318548e+21, 3.22487903e+21,
+    3.34657258e+21, 3.46826613e+21, 3.58995968e+21, 3.71165323e+21, 3.83334677e+21, 3.95504032e+21, 4.07673387e+21, 4.19842742e+21, 4.32012097e+21,
+    4.44181452e+21, 4.56350806e+21, 4.68520161e+21, 4.80689516e+21, 4.92858871e+21, 5.05028226e+21, 5.17197581e+21, 5.29366935e+21, 5.41536290e+21,
+    5.53705645e+21, 5.65875000e+21, 5.78044355e+21, 5.90213710e+21, 6.02383065e+21, 6.14552419e+21, 6.26721774e+21, 6.38891129e+21, 6.51060484e+21,
+    6.63229839e+21, 6.75399194e+21, 6.87568548e+21, 6.99737903e+21, 7.11907258e+21, 7.24076613e+21, 7.36245968e+21, 7.48415323e+21, 7.60584677e+21,
+    7.72754032e+21, 7.84923387e+21, 7.97092742e+21, 8.09262097e+21, 8.21431452e+21, 8.33600806e+21, 8.45770161e+21, 8.57939516e+21, 8.70108871e+21,
+    8.82278226e+21, 8.94447581e+21, 9.06616935e+21, 9.18786290e+21, 9.30955645e+21, 9.43125000e+21, 9.55294355e+21, 9.67463710e+21, 9.79633065e+21,
+    9.91802419e+21, 1.00397177e+22, 1.01614113e+22, 1.02831048e+22, 1.04047984e+22, 1.05264919e+22, 1.06481855e+22, 1.07698790e+22, 1.08915726e+22,
+    1.10132661e+22, 1.11349597e+22, 1.12566532e+22, 1.13783468e+22, 1.15000403e+22, 1.16217339e+22, 1.17434274e+22, 1.18651210e+22, 1.19868145e+22,
+    1.21085081e+22};
```

Comment:
It might make sense to make each of these a variable that is read in using `amrex::ParmParse`. You can do it like this inside `problem_main()`:
```
amrex::ParmParse pp("metal_problem");
std::vector<double> z_vec, logphi_vec, logg_vec;
pp.queryarr("z_data", z_vec);
pp.queryarr("logphi_data", logphi_vec);
pp.queryarr("logg_data", logg_vec);
```

Reply:
Then in the `metal_problem_regression.in` file, you have three lines like this:
```
metal_problem.z_data = 1.000e2 1.000e2 1.000e2
metal_problem.logphi_data = 1.000e2 1.000e2 1.000e2
metal_problem.logg_data =1.000e2 1.000e2 1.000e2
```
where instead of the values I've put above, you just copy and paste the real values from the arrays.

Reply:
This can go in another PR. I'll resolve this for now.